src/tokens/mainnet.json

```diff
@@ -1,151 +1,459 @@
 {
   "tokens": [
     {
-      "address": "0x9b6761bf2397Bb5a6624a856cC84A3A14Dcd3fe5",
+      "address": "0x656b95E550C07a9ffe548bd4085c72418Ceb1dba",
       "decimals": 18,
-      "image": "ibera.svg",
-      "symbol": "iBERA",
-      "tags": ["featured"]
+      "image": "bgt.svg",
+      "name": "BGT",
+      "symbol": "BGT"
     },
     {
       "address": "0xac03CABA51e17c86c921E1f6CBFBdC91F8BB2E6b",
       "decimals": 18,
       "image": "ibgt.svg",
+      "name": "iBGT",
+      "protocol": "infrared",
       "symbol": "iBGT",
-      "tags": ["featured"]
+      "type": "unknown",
+      "underlyingTokens": ["0x656b95E550C07a9ffe548bd4085c72418Ceb1dba"]
```

Comment:
To correspond the BE version, please update it to:
```
"type": "vault",
"underlyingTokens": ["0xac03caba51e17c86c921e1f6cbfbdc91f8bb2e6b"] // iBGT
```

Reply:
The `underlyingTokens` is still  `0x656b95E550C07a9ffe548bd4085c72418Ceb1dba`. But should be `0xac03caba51e17c86c921e1f6cbfbdc91f8bb2e6b`.